<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="frame_splitter">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Export session</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_start_export">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Launch</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_resume_export">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Resume</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_stop_export">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chk_loop_export">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Loop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_timer_export">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00:00:00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="pb_export">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_export">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Total 0 Failure 0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QSplitter" name="top_splitter">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPlainTextEdit" name="txt_error_export">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Error output window</string>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="txt_export">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Log output window</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Import session</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSplitter" name="bottom_splitter">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPlainTextEdit" name="txt_error_import">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Error output window</string>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="txt_import">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Log output window</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_import">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Total 0 Failure 0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_start_import">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Launch</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_resume_import">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Resume</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_stop_import">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chk_loop_import">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Loop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_timer_import">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00:00:00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="pb_import">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
